RCaller/src/test/java/com/github/rcaller/RunOnlineTest.java

```diff
@@ -10,8 +10,8 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
+import static java.lang.System.currentTimeMillis;
+import static org.junit.Assert.*;
```

Comment:
our static analysis engine detects that your pull request includes a change from an explicit import to a wildcard import. According to the best practice (http://www.javadude.com/articles/importondemandisevil.html), import star (import on demand) is a bad practice, wildcard import can sometimes cause name conflicts and ambiguities.

Reply:
@hola-holo That was IDE autoreplacement, sorry!

Reply:
Fixed in https://github.com/jbytecode/rcaller/pull/38